config/routes.rb

```diff
@@ -45,6 +45,8 @@
       as: :openid_connect_configuration
 
   get '/account' => 'accounts#show'
+  get '/account/reactivate/start' => 'reactivate_account#index', as: :manage_reactivate_account
+  put 'account/reactivate/start' => 'reactivate_account#update'
```

Comment:
yeah, I thought about moving them down, but it felt weird since logically `start`is going to happen first. Our convention is more about alphabetizing, I'll change it